### PR TITLE
Normalize Infrastructures description

### PR DESCRIPTION
**Pull request by AlexFalessi**

### CODE REVIEW

**Review by alijawadfahs — APPROVED** [review]

**Review by alijawadfahs — APPROVED** [review]

File: infrastructures/infrastructure-aws-ec2/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AWSEC2Infrastructure.java [hunk 1]
```diff
@@ -590,7 +590,8 @@ public void shutDown() {
      */
     @Override
     public String toString() {
-        return getDescription();
+        return "AWSEC2Infrastructure, numberOfInstances: [" + numberOfInstances + "], " +
```
Comment (fviale) [inline infrastructures/infrastructure-aws-ec2/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AWSEC2Infrastructure.java:593]: better use this.getClass().getSimpleName() and also String.format

File: infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java [hunk 2]
```diff
@@ -520,15 +520,21 @@ public void notifyAcquiredNode(Node node) throws RMException {
...
+        return "Handles nodes from Microsoft Azure.";
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
     public String toString() {
-        return getDescription();
+        return String.format("AzureInfrastructure numberOfInstances: [%s], numberOfNodesPerInstance: [%s], imageOSType: [%s], vmSizeType: [%s], region: [%s], maxBudget: [%s]",
```
Comment (fviale) [inline infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java:531]: better use this.getClass().getSimpleName()

File: infrastructures/infrastructure-gce/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/GCEInfrastructure.java [hunk 3]
```diff
@@ -440,15 +440,18 @@ public void shutDown() {
...
+        return "Handles nodes from the Google Compute Engine.";
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
     public String toString() {
-        return getDescription();
+        return String.format("GCEInfrastructure numberOfNodesPerInstance: [%s], totalNumberOfInstances: [%s], region: [%s]",
```
Comment (fviale) [inline infrastructures/infrastructure-gce/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/GCEInfrastructure.java:451]: better use this.getClass().getSimpleName()

File: infrastructures/infrastructure-openstack/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/OpenstackInfrastructure.java [hunk 4]
```diff
@@ -283,15 +283,20 @@ public void notifyAcquiredNode(Node node) throws RMException {
...
+        return "Handles ProActive nodes using Nova compute service of Openstack Cloud.";
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
     public String toString() {
-        return getDescription();
+        return String.format("OpenStackInfrastructure numberOfInstances: [%s], numberOfNodesPerInstance: [%s], image: [%s], flavor: [%s], region: [%s]",
```
Comment (fviale) [inline infrastructures/infrastructure-openstack/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/OpenstackInfrastructure.java:294]: better use this.getClass().getSimpleName()

File: infrastructures/infrastructure-vmware/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/VMWareInfrastructure.java [hunk 5]
```diff
@@ -268,15 +268,18 @@ public void notifyAcquiredNode(Node node) throws RMException {
...
+        return "Handles nodes of VMware Cloud.";
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
     public String toString() {
-        return getDescription();
+        return String.format("VMWareInfrastructure numberOfInstances: [%s], numberOfNodesPerInstance: [%s], image: [%s]",
```
Comment (fviale) [inline infrastructures/infrastructure-vmware/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/VMWareInfrastructure.java:279]: better use this.getClass().getSimpleName()

File: infrastructures/infrastructure-winhpc/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/WinHPCInfrastructure.java [hunk 6]
```diff
@@ -465,7 +465,12 @@ public void run() {
 
     @Override
     public String getDescription() {
-        return "WinHPCInfrastructure";
+        return "Handles nodes from a Windows HPC server.";
+    }
+
+    @Override
+    public String toString() {
+        return "WinHPCInfrastructure, maxNodes: [" + maxNodes + "], timeout: [" + timeout + "]";
```
Comment (fviale) [inline infrastructures/infrastructure-winhpc/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/WinHPCInfrastructure.java:473]: better use this.getClass().getSimpleName()

**Review by fviale — APPROVED** [review]

**Comment by fviale** [comment]
Retest this please

File: infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java (hunk 2 above)
Comment (AlexFalessi) [reply infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java:531]: Same remark for this project. Cannot use this.getClass().getSimpleName() as it ads unwanted characters

**Comment by AlexFalessi** [comment]
Retest this please

**Comment by fviale** [comment]
Retest this please

**Comment by fviale** [comment]
Retest this please

**Pull request merged**